.github/workflows/release.yml

```
@@ -120,4 +120,4 @@ jobs:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: target/${{ matrix.target }}/release/boa${{ matrix.binary_extension }}
           asset_name: boa-${{ matrix.target }}${{ matrix.binary_extension }}
-          tag: ${{ github.ref }}
+          tag: ${{ github.ref }}
```

whitespace :policeman: missing newline

```suggestion
          tag: ${{ github.ref }}

```